.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -97,17 +127,27 @@ jobs:
         sed -i '' "s|.*using python.*|$python_bin_include_lib|" project-config.jam
         cd /tmp/build/boost_${BOOST_VERSION_UNDERSCORES}
         ./b2 install -j4 -a cxxflags="-std=c++17" --prefix=${GITHUB_WORKSPACE}/${INSTALL_DIR}
+
     - name: Build dakota
       if: steps.cache-package.outputs.cache-hit != 'true'
       run: |
+        if [ -d /tmp/build-env ]; then
+          source /tmp/build-env/bin/activate
+        fi
+        python --version
+
+        export PYTHON_EXECUTABLE=$(which python3)
```

Comment:
I think I wrote it, but does it maybe make sense to just write `python` instead of `python3` here, if it has the same effect? Since we are sourcing the venv which is python3 anyway.